central/cluster/service/service_impl.go

```diff
@@ -141,6 +141,11 @@ func (s *serviceImpl) getClusterRetentionInfo(ctx context.Context, cluster *stor
 		return nil, nil
 	}
 
+	// If retention is disabled, there is no "days remaining" calculation to be done.
```

Comment:
Can we also short circuit the `getClusterIDToRetentionInfoMap` function when retentionDays == 0 ?

Reply:
I don't think it makes a massive difference since the beginning of this method doesn't really do much. And it's cleaner than leaking the config outside of this method. Thoughts?